content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md

```diff
@@ -31,11 +31,11 @@ in the Kubernetes source directory for a canonical example.
 
 Package your scheduler binary into a container image. For the purposes of this example,
 you can use the default scheduler (kube-scheduler) as your second scheduler.
-Clone the [Kubernetes source code from GitHub](https://github.com/kubernetes/kubernetes)
+Clone the [Kubernetes source code from GitHub](https://github.com/kubernetes/kubernetes) with the exact same branch as per the kubernetes version in use
 and build the source.
 
 ```shell
-git clone https://github.com/kubernetes/kubernetes.git
+git clone --branch=`kubectl version --short | grep 'Server' | awk '{print $3}'` https://github.com/kubernetes/kubernetes.git
```

Comment:
`kubectl version --short`, the --short is deprecated
not all users will have `awk` installed